frontend/src/components/Toast.js

```diff
@@ -48,11 +48,10 @@ export function ToastManager() {
                   fontSize="md"
                   fontWeight="medium"
                   flexShrink={1}
-                  color="darkText"
                 >
                   {title}
                 </Text>
-                <Text color="text.light" px="6" color="darkText">
+                <Text color="text.light" px="6">
                   {description}
                 </Text>
                 <Divider bg="transparent" thickness="2" mx="2" />
```

Comment:
It looks like there was a redundant `color` prop in the first `Text` component that has been removed, which is a good improvement. However, in the second `Text` component, the `color` prop is still set to "text.light" despite the comment removal of the previous "darkText" property. This may or may not be intentional, depending on the design requirements.

Other than the issue with the `color` prop, the code patch seems to be fine.